armi/reactor/tests/test_components.py

```diff
@@ -1295,6 +1440,13 @@ def test_getBoundingCircleOuterDiameter(self):
 
 
 class TestDifferentialRadialSegment(TestShapedComponent):
+    """Test differential radial segment.
+
+    .. test:: Compute properties of radial segment that can be updated
+        :id: T_ARMI_COMP_FLUID1
+        :tests: R_ARMI_COMP_FLUID
+    """
+
```

Comment:
Delete this one.  As stated in your `impl` crumb, this just isn't the right place.

Thanks!